spec/forms/user_piv_cac_verification_form_spec.rb

```diff
@@ -43,15 +46,18 @@
         let(:user) { create(:user, :with_piv_or_cac) }
 
         it 'returns FormResponse with success: false' do
-          result = instance_double(FormResponse)
-
-          expect(FormResponse).to receive(:new).
-            with(success: false, errors: { type: 'user.piv_cac_mismatch' },
-                 extra: { multi_factor_auth_method: 'piv_cac',
-                          multi_factor_auth_method_created_at: nil,
-                          piv_cac_configuration_id: nil,
-                          key_id: nil }).and_return(result)
-          expect(form.submit).to eq result
+          result = form.submit
+
+          expect(result.to_h).to eq(
+            {
```

Comment:
We could remove the curly's here?

Reply:
unfortunately the curly brackets are required, otherwise the equals check fails with:

```
         expected: ({:errors=>{}, :multi_factor_auth_method=>"backup_code", :multi_factor_auth_method_created_at=>2023-07-12 08:20:01.000000000 -0700, :success=>true}) (keyword arguments)
              got: ({:errors=>{}, :multi_factor_auth_method=>"backup_code", :multi_factor_auth_method_created_at=>2023-07-12 08:20:01.000000000 -0700, :success=>true}) (options hash)
 ```

Reply:
That's odd, I just tried this locally and it passes for me?

```diff
$ git diff
diff --git a/spec/forms/user_piv_cac_verification_form_spec.rb b/spec/forms/user_piv_cac_verification_form_spec.rb
index dd8b14420..3efe2f42d 100644
--- a/spec/forms/user_piv_cac_verification_form_spec.rb
+++ b/spec/forms/user_piv_cac_verification_form_spec.rb
@@ -49,14 +49,12 @@ RSpec.describe UserPivCacVerificationForm do
           result = form.submit
 
           expect(result.to_h).to eq(
-            {
-              success: false,
-              errors: { type: 'user.piv_cac_mismatch' },
-              multi_factor_auth_method: 'piv_cac',
-              multi_factor_auth_method_created_at: nil,
-              piv_cac_configuration_id: nil,
-              key_id: nil,
-            },
+            success: false,
+            errors: { type: 'user.piv_cac_mismatch' },
+            multi_factor_auth_method: 'piv_cac',
+            multi_factor_auth_method_created_at: nil,
+            piv_cac_configuration_id: nil,
+            key_id: nil,
           )
           expect(form.error_type).to eq 'user.piv_cac_mismatch'
         end
```

```
$ rspec spec/forms/user_piv_cac_verification_form_spec.rb:51
[...]
1 example, 0 failures
```

Reply:
I removed the curly brackets from this file, but can't from the controller spec since it's the latter form.